Server/ReasnAPI/ReasnAPI/Services/CommentService.cs

```diff
@@ -1,16 +1,103 @@
-﻿using ReasnAPI.Models.Database;
-
-namespace ReasnAPI.Services {
-    public class CommentService (ReasnContext context) {
-        private readonly ReasnContext _context = context;
-
-        /* TODO: Create following functions for this class
-         * create
-         * update
-         * delete
-         * get by ID
-         * get list by filter
-         * get all
-         */
-    }
-}
+using ReasnAPI.Mappers;
+using ReasnAPI.Models.Database;
+using ReasnAPI.Models.DTOs;
+using ReasnAPI.Services.Exceptions;
+using System.Linq.Expressions;
+
+namespace ReasnAPI.Services;
+
+public class CommentService(ReasnContext context)
+{
+    public CommentDto? CreateComment(CommentDto? commentDto)
+    {
+        ArgumentNullException.ThrowIfNull(commentDto);
+
+        context.Comments.Add(commentDto.ToEntity());
+        context.SaveChanges();
+
+        return commentDto;
+    }

```

Comment:
Why do you require from higher layer to handle CreatedAt field? This method and service is responsible for handling that.